backend/app/usecase/changelog/changelog.go

```diff
@@ -43,6 +46,12 @@ func (p Persist) GetChangeLog() ([]entity.Change, error) {
 	return p.changeLogRepo.GetChangeLog()
 }
 
+// GetLastViewedAt retrieves the last time the user viewed the change log
+func (p Persist) GetLastViewedAt() *time.Time {
+	now := p.timer.Now()
```

Comment:
Let's add a TODO here so we can implement this feature in the next PR!
TODO format: 
```
// TODO(issue#your_issue_number): fetch the last time the user viewed the change log from persistent storage.
```